ui/src/components/yaml/index.tsx

```diff
@@ -254,6 +255,8 @@
       setStreaming(false)
```

Comment:
This line correctly uses optional chaining to safely access the `getBoundingClientRect()` method on `contentRef.current`. The variable `contentToTopHeight` will hold the top position of the element referenced by `contentRef`.